e2e/test_options.go

```diff
@@ -21,23 +21,51 @@ import (
 	"flag"
 
 	"github.com/ethereum/go-ethereum/ethclient"
-	"github.com/mysteriumnetwork/node/tequilapi/client"
+	tequilapi_client "github.com/mysteriumnetwork/node/tequilapi/client"
 	"github.com/mysteriumnetwork/payments/cli/helpers"
 )
 
-var tequilaHost = flag.String("tequila.host", "localhost", "Specify tequila host for e2e tests")
-var tequilaPort = flag.Int("tequila.port", 4050, "Specify tequila port for e2e tests")
-var ethRPC = flag.String("geth.url", "", "Eth node RPC")
+// Domain defines domain in which tequilAPI tequilapi_client will be executed
+type Domain int
 
-func newTequilaClient() *client.Client {
-	return client.NewClient(*tequilaHost, *tequilaPort)
+// Possible domain constants
+const (
+	Consumer Domain = 0
+	Provider Domain = 1
+)
+
+// Flags to run a test
+var (
+	ehtRpcUrl = flag.String("geth.url", "http://localhost:8545", "RPC url of ethereum node")
+)
+
+// Provider flags
+var (
+	providerTequilapiHost = flag.String("provider.tequilapi-host", "localhost", "Specify Tequilapi host for provider")
+	providerTequilapiPort = flag.Int("provider.tequilapi-port", 4050, "Specify Tequilapi port for provider")
+)
+
+// Consumer flags
+var (
+	consumerTequilapiHost = flag.String("consumer.tequilapi-host", "localhost", "Specify Tequilapi host for consumer")
+	consumerTequilapiPort = flag.Int("consumer.tequilapi-port", 4050, "Specify Tequilapi port for consumer")
+)
+
+func newTequilapiClient(domain Domain) *tequilapi_client.Client {
+	switch domain {
+	case Consumer:
```

Comment:
Why we need those two constants? As I mentioned earlier, two separate functions with clear names maybe would be more readable, i.e. NewConsumerTequilApi, NewProviderTequilApi

Reply:
Made 2 different factories